django/thunderstore/account/forms.py

```diff
@@ -0,0 +1,111 @@
+import ulid2
+from django import forms
+from django.contrib.auth.models import User
```

Comment:
This should really be
```python
from django.contrib.auth import get_user_model

User = get_user_model()
```
to future proof against user model changes

Reply:
For typecheck purposes this would be alright, but it's directly also used on line 38

Reply:
I can't really tell exactly how django-stubs recommends type hinting `User`, the closest [it says](https://github.com/typeddjango/django-stubs#how-can-i-create-a-httprequest-thats-guaranteed-to-have-an-authenticated-user) is:
> where `User` is the user model specified by the `AUTH_USER_MODEL` setting.